dev/oranda.json

```diff
@@ -1,8 +1,11 @@
 {
   "homepage": "https://oranda.axo.dev",
   "readme_path": "dev/README.md",
+  "repository": "https://github.com/axodotdev/cargo-dist",
+  "artifacts": {
+    "cargo_dist": true
+  },
```

Comment:
this is for testing since cargo dist has cargo dist and we do not, will be deleted when the PR is done

Reply:
we can update this i think!